doc/onboarding/view/step3.md

```diff
@@ -0,0 +1,106 @@
+
+
+## Step 3/5 : Create a password
+
+### URI
+
+`:userID/password`
+
+
+### Models
+
+<pre>
+user = {
+	&lt;id&gt; id,
+}
+</pre>
+<pre>
+form = {
+	&lt;boolean&gt; disabled: true
+	fields: [
+		{
+		    &lt;slug&gt; name: 'password',
+		    &lt;string&gt; label: 'Password'
+		    &lt;string&gt; type: 'password'
+		    &lt;password&gt; value
+		}
+	]
+}
+</pre>
+<pre>
+step = {
+	&lt;string&gt; slug: 'set_password',
+	&lt;step&gt; value: 3/5,
+}
+</pre>
+
+
+### Getters
+
+<pre>
+    getPasswordComplexity: (state) ->
+        // Apply algorithm to know
```

Comment:
Agree with @gregorylegarec. The function to get/check the password complexity should be a part of the password component, not the view. The view will check the password complexity using this function to validate the form/view.